docs/UserGuide.md

```diff
@@ -218,66 +194,60 @@ Adds a time period where the staff is working to the staff’s schedule.
 
 Formats:
 
-`addSchedule [n/name] [day-startTime-endTime]` \
-`addSchedule [id/ID] [day-startTime-endTime]` \
-`addSchedule [n/name] [t/nonrecurr] [d/date] [startTime-endTime]` \
-`addSchedule [id/ID] [t/nonrecurr] [d/date] [startTime-endTime]`
+`addSchedule n/name d/fullDayName-shiftNumber` \
+`addSchedule i/index d/fullDayName-shiftNumber`
 
 
-* There are two ways to identify the staff to add the time period to: by their name or by their staff ID.
-* Also, there are two types of time periods that can be added - recurring and non-recurring.
-* The default is a recurring schedule which will occur every week. For the addition of the non-recurring time periods,
-  the date which the staff works has to be specified.
+* There are two ways to identify the staff to add the time period to: by their `name` or by their staff `index`.
+* The `fulldayname` field required to specify shifts are not case sensitive.
 
 Examples:
 
-`addSchedule n/Candice Mon-0800-1200` \
-`addSchedule id/12345678 t/nonrecurr d/2021-09-09 0800-1200`
+`addSchedule n/Candice d/Monday-1` \
+`addSchedule i/1234 d/tuesday-2`
 
 ### View a staff schedule : `viewSchedule`
 
 Views a specific staff’s schedule.
 
 Formats:
 
-`viewSchedlue [n/name]` \
-`viewSchedlue [id/ID]`
+`viewSchedlue n/name` \
+`viewSchedlue i/index`
 
 Examples:
 
 `viewSchedule n/Candice` \
-`viewSchedule id/12345678`
+`viewSchedule i/123`
 
 
 ### Deleting a staff schedule: `deleteSchedule`
 
-Deletes a time period from the staff schedule.  There are two ways to identify the staff to delete the time period from: by their name or by their staff ID. The deleted period must be the same as a period previously entered by the manager. If the period is a non-recurring period, the date must be specified.
+Deletes a time period from the staff schedule.  There are two ways to identify the staff to delete the time period from: by their `name` or by their staff `index`. The deleted period must be the same as a period previously entered by the manager.
 
 Formats:
 
-`deleteSchedule [n/name] [day-startTime-endTime]` \
-`deleteSchedule [id/ID] [day-startTime-endTime]`\
-`deleteSchedule [n/name] [t/nonrecurr]  [d/date] [startTime-endTime]` \
-`deleteSchedule [id/ID] [t/nonrecurr] [d/date] [startTime-endTime]`
+`deleteSchedule n/name d/fullDayName-shiftNumber` \
+`deleteSchedule i/index d/fullDayName-shiftNumber`
 
 Examples:
 
-`deleteSchedule n/Joe Mon-0800-1200` \
-`deleteSchedule id/12345678 t/nonrecurr d/2021-09-09 0800-1200`
+`deleteSchedule n/Joe d/tuesday-2` \
```

Comment:
Maybe the shift number should be seperate from the date to make clearer parsing?

Reply:
Hmm seperate as in d/tueday 2 or d/tuesday shift/2? The current implementation in my PR is parsed as string then split which is fine I think?

Reply:
I think its fine on second thought